openapi/social/social.yaml

```diff
@@ -0,0 +1,276 @@
+openapi: 3.1.0
+info:
+  title: Social
+  version: '1.0'
+  description: These APIs allow users to use Vendasta's social capabilities. All the endpoints require either the `social` or `business` scopes to be accessed.
+servers:
```

Comment:
Please copy the security schema from the platform and add your oauth scopes

Reply:
Sounds good, done 👍